pkg/operator/staticpod/controller/staticpodfallback/static_pod_fallback_condition.go

```diff
@@ -111,3 +119,12 @@ func (fd *staticPodFallbackConditionController) sync(ctx context.Context, _ fact
 
 	return nil
 }
+
+func conditionToApplyOperatorCondition(condition *operatorv1.OperatorCondition) *applyoperatorv1.OperatorConditionApplyConfiguration {
+	return applyoperatorv1.OperatorCondition().
+		WithType(condition.Type).
+		WithStatus(condition.Status).
+		WithReason(condition.Reason).
+		WithMessage(condition.Message).
```

Comment:
poor choice. This does things like make a pointer to "" instead of properly indicating nil